docker/moonbase-parachain.Dockerfile

```diff
@@ -4,24 +4,24 @@
 
 FROM phusion/baseimage:0.11
 LABEL maintainer "alan@purestake.com"
-LABEL description="this is the parachain node running Moonbase Alphanet"
+LABEL description="this is the parachain node running Moonbase"
 ARG PROFILE=release
 
 RUN mv /usr/share/ca* /tmp && \
 	rm -rf /usr/share/*  && \
```

Comment:
Can we just start with a slimmer base image?

Reply:
What do you suggest ?

Reply:
I checked and it adds 20MB overhead to the binary. (total 140MB) 
Not sure we can do a lot better

Reply:
I've heard people having success with Alpine. https://hub.docker.com/_/alpine/ It claims to be only 5MB.

I'm not really saying that our image is too big as it is. I just hadn't seen anyone manually removing stuff that comes in the base image before.